rosidl_generator_py/package.xml

```diff
@@ -12,6 +12,10 @@
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
 
   <exec_depend>rosidl_parser</exec_depend>
+  <build_depend>rosidl_generator_c</build_depend>
```

Comment:
super nitpicky, but in most package.xml files the `exec_depends` and `build_depends` are grouped together, can you do that here?